suite_dadadj.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Given that `dadadj` is the first true CAM7 physics scheme, I might vote to make a copy of this SDF called `suite_cam7.xml`.  I then might move the `suite_dadadj.xml` file to a new `test/test_sdfs` directory, as we will likely only use this particular SDF for testing purposes.

Reply:
Temporarily left a suite_dadadj.xml in the original spot for testing but will remove before commit.  Updated the rest as recommended.